base/install_modules.sh

```diff
@@ -69,3 +69,11 @@ EPICS_BASE
 install_from_github  epics-modules iocStats DEVIOCSTATS $IOCSTATS_VERSION "
 EPICS_BASE
 "
+
+download_from_github slac-epics-modules ipmiComm $IPMICOMM_VERSION
+patch -d ipmiComm -Np1 < ipmicomm.patch
+install_module ipmiComm IPMICOMM "
+EPICS_BASE
+ASYN
+DEVIOCSTATS
```

Comment:
As far as I can tell `DEVIOCSTATS` isn't needed by it either. Did you see it being required anywhere?